crates/nu-command/tests/commands/help.rs

```diff
@@ -380,3 +380,9 @@ fn help_alias_before_command() {
 
     assert!(actual.out.contains("Alias"));
 }
+
+#[test]
+fn nothing_type_annotation() {
+    let actual = nu!("def foo []: nothing -> nothing {}; help foo");
+    assert!(actual.out.contains("│ 0 │ nothing │ nothing │"));
```

Comment:
if we ever change how tables are drawn this is going to make people cry 😅 

```
def foo []: nothing -> nothing {}; help commands | where name == foo | get input_output.0.output.0
```

would get to the value directly thanks to the recent PRs

Reply:
this is a very good point :+1:

Reply:
if i did not mess up the `pipeline` call, should be more robust in cf97dc9fd :yum: